winui/Rating/Clear-rating.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Clear rating value in WinUI Rating control | Syncfusion
+description: Learn all about how to enable the clear enable feature in the Rating control the user can remove their rating or not.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# Clear rating value in WinUI Rating
+
+The Rating control provides value support to determine whether a user can remove their rating or not, by using the `IsClearEnabled` property.
```

Comment:
@Priyanka-Vijayasankar The `IsClearEnabled` property of the Rating control allows you to clear the provided rating value when the value is true.

Reply:
Updated the above changes.